projects/rocprofiler-sdk/source/bin/rocprofv3.py

```diff
@@ -437,6 +437,44 @@ def add_parser_bool_argument(gparser, *args, **kwargs):
         nargs="*",
     )
 
+    spm_options = parser.add_argument_group("Streaming Perfmon options")
+
+    add_parser_bool_argument(
+        spm_options,
+        "--spm-beta-enabled",
+        help="enable SPM; beta version",
+    )
+
+    spm_options.add_argument(
+        "--spm",
+        help=(
+            "Specify SPM events to collect(comma OR space separated in case of more than 1 counters). "
+            "Note: job will fail if entire set of counters cannot be collected in single pass"
+        ),
+        default=None,
+        nargs="*",
+    )
+
+    spm_options.add_argument(
+        "--spm-buffer-size",
+        help="SPM Buffer size",
+        default=None,
+        type=str,
+    )
+
+    spm_options.add_argument(
+        "--spm-timeout",
+        help="Timeout for SPM, in ms. Larger values may slow down the application, while smaller values may drop data. Default = 30.",
+        default=None,
+        type=int,
+    )
+
+    spm_options.add_argument(
+        "--spm-frequency",
+        help="SCLK Sample frequency, in Hz. Default: 500 KHz",
```

Comment:
I assume not many developers are familiar with the SCLK acronym, so maybe this could help.

```suggestion
        help="SCLK (shader clock) Sample frequency, in Hz. Default: 500 KHz",
```

Reply:
Done